kafka-queue/kafka_util.go

```diff
@@ -0,0 +1,140 @@
+package kafka_queue
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+func NewKafkaProducer(kafkaServer string) *kafka.Producer {
+	producer, err := kafka.NewProducer(&kafka.ConfigMap{
+		"bootstrap.servers": kafkaServer,
+		"acks":              "all"})
+	if err != nil {
+		fmt.Println("Failed to create producer due to ", err)
+		panic(err)
```

Comment:
Panicking in code will stop the program. 
In production, this service will stop working is there is an error.
Best to print the error and return.